### DISCUSSION TITLE
How do I correctly use my fork of next-auth in my project?

### DISCUSSION THREAD

**Discussion by saturation-jonathan** [nextauthjs/next-auth#6861] [Help]
Hi there,
Thanks in advance for your help :)

I made a fork of this repo and in my package.json I pointed to my fork as such in the dependencies:
`"next-auth": "https://github.com/<user_name>/next-auth.git#test_branch"`

after I `yarn install` and then `yarn dev`, I get the error:
```
error - ./src/pages/_app.tsx:8:0
Module not found: Can't resolve 'next-auth/react'
>  8 | import { SessionProvider } from "next-auth/react"
```

After looking at the next-auth folder in my node_modules it seems that none of the folders (client, core, css, jwt, next, react, ...) are inside it. Instead it looks like this:
![Screenshot 2023-03-02 at 11 46 04 AM](https://user-images.githubusercontent.com/123182162/222497582-a5cd87b5-a9e9-4d8a-8127-4b95843111dc.png)

when it should be looking like this:
![Screenshot 2023-03-02 at 11 39 46 AM](https://user-images.githubusercontent.com/123182162/222497686-3e00aa68-02be-4bb5-9e30-aebe7fb4f936.png)

Just wondering if I could get some guidance on where I went wrong here. I'm not very experienced with npm, but I did try to implement some solutions found here with no luck:

- https://www.pluralsight.com/guides/how-to-use-forked-npm-dependencies
- https://gist.github.com/stephen-peck/719ec5419c670677f94e97f7ea40ab56
- https://stackoverflow.com/questions/40528053/npm-install-and-build-of-forked-github-repo

Thanks for your help!

**Comment by Khaaz** [comment]
I have the exact same question.
